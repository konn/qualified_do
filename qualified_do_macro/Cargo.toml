[package]
name = "qualified_do_macro"
version = "0.0.0"
autotests = false
license.workspace = true
edition.workspace = true
authors.workspace = true

[lib]
proc-macro = true

[[test]]
name = "tests"
path = "tests/trybuild.rs"

[dev-dependencies]
trybuild.workspace = true
functo_rs.workspace = true
either.workspace = true

[dependencies]
syn.workspace = true
proc-macro2.workspace = true
quote.workspace = true
