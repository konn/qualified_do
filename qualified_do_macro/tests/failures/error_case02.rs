fn main() {
    use qualified_do_macro::qdo;

    qdo! {hoge::fuga {
        return 4;
        x <- Some(5)
    }}
}
