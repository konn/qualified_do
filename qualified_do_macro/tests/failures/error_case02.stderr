error: Expected an expression or return statement at the last
 --> tests/failures/error_case02.rs:6:9
  |
6 |         x <- Some(5)
  |         ^^^^^^^^^^^^
